lib/travis/build/templates/header.sh

```diff
@@ -28,6 +28,24 @@ export SHELL
 export TERM
 export USER
 
+case $(uname | tr '[A-Z]' '[a-z]') in
+  linux)
+    export TRAVIS_OS_NAME=linux
+    ;;
+  *)
+    export TRAVIS_OS_NAME=osx
```

Comment:
Can this be a match on `darwin` instead of `*`, and have `*` result in something like `TRAVIS_OS_NAME=notset`?